Dart/benchmark.js

```diff
@@ -269,7 +269,7 @@ class Benchmark {
     if (isInBrowser) {
       // In browsers, relative imports don't work since we are not in a module.
       // (`import.meta.url` is not defined.)
-      let pathname = location.pathname.match(/(.*)index\.html/)[1];
+      const pathname = location.pathname.match(/^(.*\/)(?:[^.]+(?:\.(?:[^\/]+))+)?$/)[1];
```

Comment:
It might be easier to read if you use `replace`, for example `.replace(/\/index\.html$/, '/')`

Reply:
Yeah, although I'd rather do that in a follow up since I'd like to unblock this.